src/components/GordonLimitedAvailability/index.js

```diff
@@ -0,0 +1,40 @@
+import { Grid, Card, CardContent, Button } from '@material-ui/core';
+import { Link } from 'react-router-dom';
+
+const GordonLimitedAvailability = ({ pageName, backToLocation, backToLink, availableTo }) => {
```

Comment:
You can add defaults this way:
```suggestion
const GordonLimitedAvailability = ({ pageName = '', backToLocation = 'Home', backToLink = '', availableTo = 'students' }) => {
```